docs/README.rst

```diff
@@ -250,6 +252,37 @@ You can attach multiple certificate/key pairs to the HTTPS virtual server.
 The BIG-IP device uses `TLS Server Name Indication`_ (SNI) to choose the correct certificate to present to the client; SNI allows the `Cloud Foundry`_ instance to support multiple hostnames (foo.mycf.com and bar.mycf.com).
 Some of these cert/key pairs can be wildcard (\*.mycf.com).
 
+.. _session persistence:
+
+JSESSIONID Session Persistence
+------------------------------
+
+The |cfctlr| supports session persistence on the BIG-IP system through cookies. Session persistence is enabled by default. To turn it off, set the :code:`session_persistence` :ref:`configuration parameter <cfctlr-configuration>` to "false", as shown below.
```

Comment:
The |cfctlr| enables session persistence on the BIG-IP using cookies by default.

Reply:
Thanks!